spec/features/posts/visit_post_index_spec.rb

```diff
@@ -0,0 +1,28 @@
+require "rails_helper"
+
+feature "visit posts index", %Q{
```

Comment:
Use `%` instead of `%Q`.<br>Use `%Q` only for strings that contain both single quotes and double quotes, or for dynamic strings that contain double quotes.